src/corpus/ui/ChapterViewLine.tsx

```diff
@@ -148,53 +198,99 @@ export function ChapterViewLine({
   ] = useContext(RowsContext)
   const [{ language }] = useContext(TranslationContext)
 
+  const column = useMemo(
+    () => createColumns(line.variants[variantNumber].reconstruction),
+    [line, variantNumber]
+  )
+
   const transliteration = useMemo(
     () => (
       <>
-        <LineNumber line={line} activeLine={activeLine} />
-        <LineColumns columns={columns} maxColumns={maxColumns} />
+        {variantNumber === 0 ? (
+          <LineNumber line={line} activeLine={activeLine} />
+        ) : (
+          <td style={{ textAlign: 'right' }}>
```

Comment:
Do not use inline styles.

If you prefer to use e.g. styled components, discuss with others and if agreed migrate all styles.